### PR TITLE
Related To is now an array of strings, not ids

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-import/patient_importer.rb [hunk 1]
```diff
@@ -110,11 +110,12 @@ def normalize_references(patient, entry_id_map)
 
           relations_to_add = []
           data_element.relatedTo.each do |related_to|
-            relations_to_add << entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
+            relations_to_add += entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
           end
-          data_element.relatedTo.destroy
+          # Clear out relatedTo element. We will be replacing the ids from the QRDA document with our interal ids.
```
Comment (lmd59) [inline lib/qrda-import/patient_importer.rb:115]: typo, internal

File: lib/qrda-import/patient_importer.rb [hunk 2]
```diff
@@ -110,11 +110,12 @@ def normalize_references(patient, entry_id_map)
...
+          data_element.relatedTo = []
```
Comment (lmd59) [inline lib/qrda-import/patient_importer.rb:116]: looks like these 5 lines of code could be simplified to `data_element.relatedTo = relations_to_add.map{ |r| r.to_s }` ?

File: lib/qrda-import/patient_importer.rb [hunk 3]
```diff
@@ -110,11 +110,12 @@ def normalize_references(patient, entry_id_map)
 
           relations_to_add = []
           data_element.relatedTo.each do |related_to|
-            relations_to_add << entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
+            relations_to_add += entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
```
Comment (lmd59) [inline lib/qrda-import/patient_importer.rb:113]: Not sure I understand this change since it implies that the thing from the entry_id_map goes from a singular object to an array of those objects.

Comment (dczulada) [reply lib/qrda-import/patient_importer.rb:113]: The old code resulted in an array that had arrays in it (since the values in entry_id_map are arrays).  The new code combines arrays.
